src/Data/List/Relation/Binary/Permutation/Setoid/Properties.agda

```diff
@@ -144,7 +144,7 @@ steps-respʳ ys≋xs           (trans ys↭ws ws↭zs) = cong (steps ys↭ws +_)
 ------------------------------------------------------------------------
 -- map
 
-module _ (T : Setoid b ℓ) where
+module _ {ℓ} (T : Setoid b ℓ) where
```

Comment:
Note that this is a side-fix: previously the function's co-domain (equivalence) was unnecessarily restricted to live at the same level as the domain (equivalence).